### DISCUSSION TITLE
Isn't sqlite.sql updated in sync on version upgrades?

### DISCUSSION THREAD

**Discussion by yeaze** [ccfos/nightingale#2349] [General]
之前使用v7.6版本， 最近看到升级到v7.7.2了， 我看数据库结构是有变更的，但是看了 docker/sqlite.sql 文件好像还是老的，没有更新

**Comment by 710leo** [comment]
@yeaze 升级到 7.7.2 版本之后，有什么问题吗？表结构应该会自动变更的

**Reply by yeaze** [reply]
我的意思是7.7.2源代码中，docker/initsql/a-n9e.sql 文件中的表结构和 docker/sqlite.sql 中的不一致。
 举个列子：
docker/initsql/a-n9e.sql 文件中的 `alert_rule`表新增了 datasource_queries这个字段，但是 docker/sqlite.sql 文件中的  `alert_rule` 表中并没有这个字段。

假如我使用sqlite 作为数据库，那缺少数据库字段，对功能使用上也应该是有影响的吧？
